src/deps.ts

```diff
@@ -40,13 +40,32 @@ export function toDependencies(object: {}): IDependency[]
  */
 export function parseVersion(version: string): string
 {
-  if (!/^[~^]/.test(version)) {
-    return version;
+  if (/^[~^]/.test(version)) {
```

Comment:
Can we get some comments on what this test is doing please?

Reply:
& Why we're returning here and not using the one below etc

Reply:
This is exactly the same test that existed before. Do you want me to add comments for code that already existed?

Reply:
We're returning here because it finds a valid version string starting with ^ or ~. If test fails, it assumes it will be a url string and goes on to extract the version from the url.

Reply:
My mistake, thats cool.